### PR TITLE
3 create search component theme

### PR DESCRIPTION
**Pull request by LazarevSlava**
# Changed

added theme on search

## Checklist

- [x] Linter checks
- [x] Code style checks

### CODE REVIEW

**Comment by LazarevSlava** [comment]
Sorry for the new branch, in previous i missed add theme

File: src/components/utils/_mixins.scss [hunk 1]
```diff
@@ -2,4 +2,7 @@
   background-color: $theme;
   box-shadow: 0 0 1px rgba($theme, 0.25);
   color: $white;
+  transition:
+    background-color 2s,
+    color 2s;
```
Comment (michey85) [inline src/components/utils/_mixins.scss]: I think we already discussed this before - you should mention the properties first and then set the timing once.

And 2 seconds maybe is too much.

File: src/components/Search.jsx [hunk 2]
```diff
@@ -2,9 +2,11 @@ import { useState } from "react";
 import style from "../components/Search.module.scss";
 import { useDebounce } from "../hooks/useDebounce";
 import PropTypes from "prop-types";
-
+import { useTheme } from "../hooks/themeUtils";
```
Comment (michey85) [inline src/components/Search.jsx:5]: Add an empty line between a component and imports.

File: src/components/utils/_mixins.scss (hunk 1 above)
Comment (LazarevSlava) [reply src/components/utils/_mixins.scss]: i made it but it doesnt work, if i use    background-color,
    color 2s; only color has 2s

File: src/components/Search.jsx (hunk 2 above)
Comment (LazarevSlava) [reply src/components/Search.jsx:5]: done

**Review by michey85 — APPROVED** [review]

**Pull request merged**